CONTRIBUTING.md

```diff
@@ -3,45 +3,55 @@
 We'd love to have you contribute to Loris. The first thing you should do
 before contributing is probably sign up for the [LORIS developers' mailing list](http://www.bic.mni.mcgill.ca/mailman/listinfo/loris-dev).
 
+Your next step before issuing a pull request is to review our [Coding Standards](https://github.com/aces/Loris/blob/minor/docs/CodingStandards). If you are doing front-end development you should also check out our [React guidelines](https://github.com/johnsaigle/Loris/blob/180631-Contributing/LORIS_react.README.md).
```

Comment:
@johnsaigle I think github supports relative links and in this case you would want to link to the coding standards on your own branch, not minor

Reply:
Oops accident